bundle/manifests/rhpam-kogito-operator-controller-manager_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: null
```

Comment:
isn't better remove this?

Reply:
The cluster will update, I don't think it should be a problem. The original template had this attribute set to `null`, @Kaitou786 ?

Reply:
I don't think this will create any problem, this was auto-generated. I created the [service_account.yaml](https://github.com/kiegroup/rhpam-kogito-operator/pull/6/files#diff-680955b3d8e65cb74bf159e966e912a9f845f8b598ff2716ea02977972274a5f) and didn't mentioned this field. The null was given by operator-sdk. It is same in the sample operator code that they give us: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/manifests/memcached-operator-controller-manager_v1_serviceaccount.yaml#L4